### ISSUE TITLE
dependency error on initial start 

### ISSUE THREAD

**Issue by itsnuwan** [kriasoft/react-starter-kit#1019]
### Followed below simple steps and stuck with error message 
```
git clone -o react-starter-kit -b master --single-branch  https://github.com/kriasoft/react-starter-kit.git MyApp
cd MyApp
npm install
npm start 


ReferenceError: Unknown plugin "syntax-trailing-function-commas" specified in "package.json"
```

**Comment by itsnuwan** [comment]
this resolves by updating node into version 7.2.1(latest)

**Comment by itsnuwan** [comment]
this resolves by updating node into version 7.2.1(latest)
